libs/blocks/modal/modal.css

```diff
@@ -25,8 +25,8 @@
   z-index: 103;
 }
 
-.dialog-modal.commerce-frame .fragment,
-.dialog-modal.commerce-frame .section {
+.dialog-modal.commerce-frame > .fragment,
+.dialog-modal.commerce-frame > .section {
   height: 100vh;
```

Comment:
It is better to move this rule on line 206 (into the @media (max-width: 1200px) {}), otherwise it is applied on the desktop and makes the modal content height too big (adds a scroll). I added this change in [this PR](https://github.com/adobecom/milo/pull/1946/files), but not sure which one will be merged first :)

Reply:
@mirafedas  
the `>` in the rule is still needed because otherwise they apply to all the nested fragments and breaks the page.
Since you have an iframe you didn't notice it.
Can you add `>` in yours?

Reply:
Thanks for info, I didn't know, I'll add it